node/pkg/watchers/solana/client.go

```diff
@@ -345,10 +355,13 @@ func (s *SolanaWatcher) Run(ctx context.Context) error {
 		zap.String("wsUrl", wsUrl),
 		zap.String("contract", contractAddr),
 		zap.String("rawContract", s.rawContract),
+		zap.String("shimContractAddr", s.shimContractStr),
```

Comment:
You have two variables: shimContractStr and shimContractAddr.  Here, the display text is shimContractAddr, but the variable displayed is shimContractStr.  A little confusing.

Reply:
Sorry it's confusing but it's intentional. I want to print out something human readable (which is the string). I guess I can change the tag to just "shimContract".